common/pom.xml

```diff
@@ -67,6 +67,7 @@
     <com.fasterxml.jackson.module-jackson-module-jsonSchema.version>2.9.4</com.fasterxml.jackson.module-jackson-module-jsonSchema.version>
     <com.fasterxml.jackson.module-jackson-module-jaxb-annotations.version>2.9.4</com.fasterxml.jackson.module-jackson-module-jaxb-annotations.version>
     <com.fasterxml.jackson.datatype-jackson-datatype-hibernate5.version>2.9.4</com.fasterxml.jackson.datatype-jackson-datatype-hibernate5.version>
+    <com.fasterxml.jackson.datatype-jackson-datatype-jdk8.version>2.9.4</com.fasterxml.jackson.datatype-jackson-datatype-jdk8.version>
```

Comment:
The pom.xml files are automatically updated based on changes made to the buildfile. While you're probably right, changing it will just cause it to get overwritten on the next build.